pages/utilites/aa_comp_calculator.md

```diff
@@ -18,6 +18,10 @@ Required arguments:
 Optional arguments:
 - `-o`, `--output <out_dir>` Path to user-defined output directory
   - Default: `./aa_comp_calculator_out_<M.D.Y>`
+- `--chimera` A .tsv containing a Unique ID, higher taxonomy, and lower taxonomy for each chimera within the input bootstrap files. The file contains three columns:
+  - Chimera_ID 1
```

Comment:
Should there be a 1 here?

Reply:
It doesn't matter. I put it there intentionally because that is how it is in the help menu, which is consistent with other scripts such as taxon_collapser. We can remove it if you want

Reply:
Fine the way it is then.